
Fixed the direct inclusion of kdev_t.h e.g. from userspace apps.

  - Rene Rebe <rene@rocklinux.org>

--- linux-2.6.8.1/include/linux/kdev_t.h.orig	2004-09-29 10:42:47.335008887 +0200
+++ linux-2.6.8.1/include/linux/kdev_t.h	2004-09-29 10:43:08.218675269 +0200
@@ -1,5 +1,8 @@
 #ifndef _LINUX_KDEV_T_H
 #define _LINUX_KDEV_T_H
+
+#include<linux/types.h>
+
 #ifdef __KERNEL__
 #define MINORBITS	20
 #define MINORMASK	((1U << MINORBITS) - 1)
